pkg/apps/blackduck/blackduck.go

```diff
@@ -2128,32 +2128,80 @@ func (b Blackduck) isBinaryAnalysisEnabled(bdspec *blackduckapi.BlackduckSpec) b
 	return false
 }
 
-func (b Blackduck) createPVC(name string, requestedSize string, defaultSize string, storageclass string, accessMode horizonapi.PVCAccessModeType, label map[string]string, blackduck *blackduckapi.Blackduck) *components.PersistentVolumeClaim {
+func GenPVC(blackDuck blackduckapi.Blackduck, defaultPVC map[string]string) ([]*components.PersistentVolumeClaim, error) {
+	var pvcs []*components.PersistentVolumeClaim
```

Comment:
@rrati @yashbhutwala not part of this PR.. Probably we should create a common method to support for all apps. We need to make all the app consistent to get PVC storage class, claim size of each PVC, volume name associating to each PVC. so that the customer will not be confused. I think, it is easy to be done.

Reply:
A common means to define/generate PVCs is a good path.  We'll need that as we add more apps to the operator.